resources/views/profile/logout-other-browser-sessions-form.blade.php

```diff
@@ -1,99 +1,182 @@
-<x-jet-action-section>
-    {{-- TODO: rewrite to our components --}}
-    <x-slot name="title">
-        @lang('ui::forms.logout-sessions.title')
-    </x-slot>
-
-    <x-slot name="description">
-        @lang('ui::forms.logout-sessions.description')
-    </x-slot>
-
-    <x-slot name="content">
-        <div class="max-w-xl text-sm text-gray-600">
+<div dusk="logout-other-browser-sessions">
+    <div class="flex flex-col">
+        <span class="header-4">@lang('ui::forms.logout-sessions.title')</span>
+        <span class="mt-4">@lang('ui::forms.logout-sessions.description')</span>
+        <x-ark-alert class="mt-8">
             @lang('ui::forms.logout-sessions.content')
-        </div>
+        </x-ark-alert>
+    </div>
 
-        @if (count($this->sessions) > 0)
-            <div class="mt-5 space-y-6">
-                <!-- Other Browser Sessions -->
-                @foreach ($this->sessions as $session)
-                    <div class="flex items-center">
-                        <div>
-                            @if ($session->agent->isDesktop())
-                                <svg fill="none" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" viewBox="0 0 24 24" stroke="currentColor" class="w-8 h-8 text-gray-500">
-                                    <path d="M9.75 17L9 20l-1 1h8l-1-1-.75-3M3 13h18M5 17h14a2 2 0 002-2V5a2 2 0 00-2-2H5a2 2 0 00-2 2v10a2 2 0 002 2z"></path>
-                                </svg>
+    @if (count($this->sessions) > 0)
+        <table class="hidden mt-8 w-full text-left table-auto md:table">
```

Comment:
the foundation library has table styling and components, can you use those instead?

Reply:
Done